.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,42 +14,45 @@
 
 ## Proposed change
 <!-- 
-    Describe the big picture of your changes here to communicate to the
-    maintainers why we should accept this pull request. If it fixes a bug
-    or resolves a feature request, be sure to link to that issue in the 
-    additional information section.
+  Describe the big picture of your changes here to communicate to the
+  maintainers why we should accept this pull request. If it fixes a bug
+  or resolves a feature request, be sure to link to that issue in the
+  additional information section.
 -->
 
 
 ## Type of change
 <!--
-    What types of changes does your PR introduce to our documention/website?
-    Put an `x` in the boxes that apply. You can also fill these out after
-    creating the PR.
+  What types of changes does your PR introduce to our Home Assistant?
```

Comment:
What type of changes does your PR introduce to Home Assistant?

Reply:
🤦‍♂ I picked a simple job for tonight...

Reply:
Also adjusted `changes` -> `change`, since: Single box 👍